mineinabyss-features/src/main/kotlin/com/mineinabyss/anticheese/AntiCheeseListener.kt

```diff
@@ -45,6 +48,24 @@ class AntiCheeseListener : Listener {
         }
     }
 
+    @EventHandler
+    fun EntityMoveEvent.onRidableMobEnterWater() {
+        val mount = entity.toModelEntity()?.mountHandler ?: return
+        if (entity.toGearyOrNull() == null) return
+        if (!isPluginEnabled("BoneHurtingJuice") || entity.fallDistance < BoneHurtConfig.data.minFallDist) return
```

Comment:
Might be better to put this up since these calculations are a lot faster than going through ModelEngine probably

Reply:
probably, ideally BHJ would just work for entities aswell. In that case its probably better to have it in there since these are very not perfect